### ISSUE TITLE
A kind request for clarification

### ISSUE THREAD

**Issue by JosephKJ** [YangNaruto/latent-energy-transport#2]
Hi Yang Zhao (@YangNaruto),

Thanks again for your amazing work and elegant code base. It would be very kind of you if you could help me clarify one aspect of the loss formulation. 

The equation 2 reads as follows:
![image](https://user-images.githubusercontent.com/4231550/128631040-5ad2e9c9-155b-41b1-852e-2d7b7e36528a.png)

This nicely translates to:

`loss = -(target_energy - source_energy).mean()`  in code: [[1]](https://github.com/YangNaruto/latent-energy-transport/blob/master/Beta-VAE/translation.py#L265), [[2]](https://github.com/YangNaruto/latent-energy-transport/blob/master/synthetic/synthetic.py#L194). 

The second part of equation 2 is an expectation over samples from the EBM, which are sampled using Langevin dynamics as follows: 

```
z_src_q = langvin_sampler(ebm, z_src)
source_energy = ebm(z_src_q)
```

Why do we start the Langevin sampling from the source domain's z (z_src)? I will explain what indeed confuses me. Equation 2 basically tries to learn the energy manifold of the target domain. Why do we use the source domain information to learn the target energy distribution?

**Comment by JosephKJ** [comment]
I believe those would act as negative samples to learn the target distribution. Can you please confirm whether my understanding is correct?

**Comment by YangNaruto** [comment]
Hi Joseph,

Sorry for the late reply and thanks for your interest. yep, you are right. More specifically,

The reason why Langevin sampling starts from the source domain: the goal of image translation is to translate the source domain's distribution to the target domain's distribution. So we need the source samples to serve as the initial samples of the Langevin sampler.

**Comment by JosephKJ** [comment]
Thank you very much @YangNaruto! This certainly clarifies.
